recipes/pestpp/meta.yaml

```
@@ -0,0 +1,35 @@
+{% set version = "5.1.6" %}
+
+package:
+  name: pestpp
+  version: {{ version }}
+
+source:
+  url: https://github.com/usgs/pestpp/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 1f1d8b4a50dfff5e0ce9f5c4d48c037aa8345c99cea3768509b9f1cd473fc53d
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make   # [not win]
+    - ninja  # [win]
+
+test:
+  commands:
+    - ldd $(which pestpp-ies)  # [linux]
+    - otool -L $(which pestpp-ies)  # [osx]
+    - where pestpp-ies  # [win]
```

```suggestion
  commands:
    - ldd $(which pestpp-ies)  # [linux]
    - otool -L $(which pestpp-ies)  # [osx]
    - where pestpp-ies  # [win]
```
Typically, we use existence testing to check that libs, headers, binaries, and CMake artifacts are installed where expected. Do that instead.